asynq/debug.py

```diff
@@ -20,6 +20,9 @@
 import traceback
 import logging
 from sys import stderr, stdout
+from pygments import highlight # pygments loading behavior requires this style import
```

Comment:
This needs to be added as a dependency in setup.py and potentially elsewhere (maybe requirements.txt and the wheel building script).